godotenv.go

```diff
@@ -37,10 +37,8 @@ func Load(filenames ...string) (err error) {
 
 	for _, filename := range filenames {
 		err = loadFile(filename)
-		if err != nil {
-			return // return early on a spazout
-		}
 	}
+	// return the last load error
 	return
```

Comment:
The error could be completely ignored if the first load fails and the second succeeds.
It is fine for `.env.local`, but I am not sure it is really a wanted behavior if called with something like

```
godotenv.Load("filenumberone.env", "filenumbertwo.env")
```